src/main/java/com/beerjournal/breweriana/item/rating/RatingController.java

```diff
@@ -0,0 +1,37 @@
+package com.beerjournal.breweriana.item.rating;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/api/ratings")
+@RequiredArgsConstructor
+public class RatingController {
+
+    private final RatingService ratingService;
+
+    @GetMapping("item/{itemId}")
```

Comment:
I meant that you can place `/api/ratings` in class annotation and method annotations are shorter, you know, file size is smaller etc :D
Ok, I will probably commit the last path you proposed (`/api/ratings?itemId=id`).